provider/FeatureCollectionMDS.json

```diff
@@ -0,0 +1,95 @@
+{
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "$id": "https://github.com/CityofLosAngeles/mobility-data-specification/blob/master/FeatureCollectionMDS.json",
+  "title": "GeoJSON FeatureCollection (MDS spin)",
+  "type": "object",
+  "required": [
+    "type",
+    "features"
+  ],
+  "properties": {
+    "type": {
+      "type": "string",
+      "emum": [
+        "FeatureCollection"
+      ]
+    },
+    "features": {
+      "type": "array",
+      "items": {
+        "title": "GeoJSON Feature",
+        "type": "object",
+        "required": [
+          "type",
+          "properties",
+          "geometry"
+        ],
+        "properties": {
+          "type": {
+            "type": "string",
+            "emum": [
+              "Feature"
+            ]
+          },
+          "properties": {
+            "type": "object",
+            "required": [
+              "timestamp"
+            ],
+            "properties": {
+              "timestamp": {
+                "type": "number",
+                "minimum": 0.0
+              }
+            }
+          },
+          "geometry": {
+            "title": "GeoJSON Point",
+            "type": "object",
+            "required": [
+              "type",
+              "coordinates"
+            ],
+            "properties": {
+              "type": {
+                "type": "string",
+                "enum": [
+                  "Point"
+                ]
+              },
+              "coordinates": {
+                "type": "array",
+                "minItems": 2,
+                "items": {
+                  "type": "number"
+                }
+              },
+              "bbox": {
+                "type": "array",
+                "minItems": 4,
+                "items": {
+                  "type": "number"
+                }
+              }
+            }
+          },
+          "bbox": {
+            "type": "array",
+            "minItems": 4,
+            "items": {
+              "type": "number"
+            }
+          }
+        }
+      },
+      "minItems": 2
+    },
+    "bbox": {
+      "type": "array",
+      "minItems": 4,
+      "items": {
+        "type": "number"
+      }
+    }
+  }
+}
```

Comment:
On a quick glance this looks good too... it covers the case of 1 single point, right?

Came across [this discussion on SO](https://stackoverflow.com/questions/41826959/how-to-reference-json-schema-definition-from-another-schema), which is also related to geometry as it turns out. I guess I *hoped* there would be an "official" `GeoJSON` spec somewhere that we could just reference in, rather than re-implement it like this. Is that hoping for too much?

Reply:
My reading of the spec was that it should have at least two points, so I tried to have that reflected here. Happy to change it to one if that is more correct.

There is an official GeoJSON spec for this, to be found [here](http://geojson.org/schema/FeatureCollection.json). Unfortunately, JSON schemas don't have a good story for extending interfaces (rather than composing them). So for this, I wrote a python script that does pull the official schema, and then patches it up to conform to the MDS narrative.

I waffled on whether that script was appropriate to include here. What do you think?

Reply:
ooooo I like that... interesting! I'm curious to see the script in any case. So the issue here is -- we can't use the official spec *and* require the `accuracy` properties or other aspects custom for MDS. Hence the script to combine. 

On the question of one point vs. two -- I was hoping we could address CityofLosAngeles/mobility-data-specification#74 via JSON Schema, and so if we had Point as a thing on its own (with a single set of coordinates) it could also be used for the `event_location` field on `status_change` events instead of the wonky "GPS string" we have now.

Reply:
Here is the script I used for generating the spin on `FeatureCollection`. It's short, so I'll inline it:
```python
import json
import requests

# Get the canonical FeatureCollection
r = requests.get("http://geojson.org/schema/FeatureCollection.json")
feature_collection = r.json()

# Modify some metadata
feature_collection["$schema"] = "http://json-schema.org/draft-06/schema#"
feature_collection["$id"] = "https://github.com/CityofLosAngeles/mobility-data-specification/blob/master/FeatureCollectionMDS.json"
feature_collection["title"] = "GeoJSON FeatureCollection (MDS spin)"

features = feature_collection["properties"]["features"]

# Only accept Points in the FeatureCollection
geometry = features["items"]["properties"]["geometry"]
point = next(g for g in geometry["oneOf"] if (g.get("title") == "GeoJSON Point"))
features["items"]["properties"]["geometry"] = point

# Force the FeatureCollection to at least have a start and an end.
feature_collection["properties"]["features"]["minItems"] = 2

# Add a required timestamp property
properties = features["items"]["properties"]["properties"]
properties["required"] = ["timestamp"]
properties["properties"] = {
        "timestamp": {
            "type": "number",
            "minimum": 0.0
            }
        }

with open('FeatureCollectionMDS.json', 'w') as out:
    out.write(json.dumps(feature_collection, indent=2))
```

Reply:
Really cool, I love how straightforward this is in python!

How about the question of addressing that issue I link above - and making `event_location` more fully specified as a GeoJSON Point?

Reply:
I think that's reasonable. Should `event_location` be a straight GeoJSON point, or do you think we should modify it in some way? If the former, we could directly reference the URL (though maybe we would want validation to also work in a setting without internet access).

Reply:
Regarding differences with the actual spec: it seems that the GeoJSON spec for primitive points, polygons, etc is intentially somewhat limited. If we want to add additional properties then it becomes a `Feature` (hence the reason that `FeatureCollection` is used here).

Reply:
I think you were right, it should be a GeoJSON `Feature` so we can add the additional properties (e.g. `accuracy`) if needed. Then the `route` is a `FeatureCollection` that is an array of this MDS-specific `Feature`, does that make sense?